app/assets/javascripts/templates/bulk_buy_modal.html.haml

```diff
@@ -2,12 +2,21 @@
   .columns.small-12
     %h3{"ng-bind" => "::variant.extended_name"}
 
-.row.variant-bulk-buy-price-summary
-  .columns.small-6
-    .variant-unit {{ ::variant.unit_to_display }}
-  .columns.small-6
+.flex.variant-bulk-buy-price-summary{style: "justify-content: space-around;"}
+  .variant-unit
+    {{ ::variant.unit_to_display }}
+  .div
     {{ variant.line_item.total_price | localizeCurrency }}
 
+  .unit-price{"ng-if": "showunitprice"}
```

Comment:
it'd be great if we could make this argument more readable like `show_unit_price`. There's quite some code the reader needs to go through so some help is more than welcome.

Reply:
I had some troubles with this. I tried camelCase but it failed. I will try with snake case.

Reply:
:see_no_evil: then I clearly don't know how this works.

Reply:
I've tested with `snake_case`... it fails.
The only solution I found is to mix both... 
Use `showunitprice` and `show_unit_price`...
See commit: a8c9f87dc